### PR TITLE
DEPENDENCY: upgrade to ember-cli v2.18

### PR DESCRIPTION
**Pull request by toranb**
long time coming ...

### CODE REVIEW

**Comment by toranb** [comment]
@williamsbdev no rush/ nothing much to look at here anyway ;)

I wasn't sure if you still had client code using this library - if so I'm looking into the `_targetObject` issue mentioned by Ben in #24 (from late 2016)

File: bower.json [hunk 1]
```diff
@@ -1,10 +1,6 @@
 {
   "name": "ember-cli-simple-validation",
   "dependencies": {
-    "ember": "1.13.13",
-    "ember-cli-shims": "0.1.0",
-    "ember-cli-test-loader": "0.2.2",
-    "ember-qunit-notifications": "0.1.0",
-    "jquery": "~2.1.4"
+    "ember": "1.13.13"
```
Comment (williamsbdev) [inline bower.json:4]: Is bower.json still required?

**Review by williamsbdev — APPROVED** [review]
@toranb - I no longer have a production app using this addon. Upon a quick review, this looks good to me. I don’t see anything that would cause alarm. If there is a regression, let me know and I can help troubleshoot/triage/fix.

File: bower.json (hunk 1 above)
Comment (toranb) [reply bower.json:4]: The addon’s dummy app was built with ArrayController so I’ve avoided the work to update it beyond ember 1.13.13 and last I checked bower is required to install this older version of the framework with ember cli

Comment (williamsbdev) [reply bower.json:4]: Gotcha. Sounds good.

**Comment by toranb** [comment]
@williamsbdev  merging this but learned today that `visible` doesn't recompute in ember >= 2.13+ so the glimmer VM 0.22 upgrade broke something related to this. Spent a good chunk of time today just trying to understand what broke and its clear the project dummy app running < ember 2 only adds to the friction.

https://github.com/toranb/ember-pure-components-example

^ that is what I've used so far to confirm ember 2.12 is fine/ but 2.13 breaks. At this point it's clear a large investment is needed to keep this addon alive so I'm planning to let the community take over (assuming anyone is using it). I'm not using this myself anymore and if you aren't its likely this addon needs to be "retired" in place of something better like changeset validations

**Comment by williamsbdev** [comment]
Nice! Good work on this.

**Comment by williamsbdev** [comment]
@toranb - I am not currently using this on any of my projects. Thanks for the heads on the support of this project.

**Pull request merged**